### DISCUSSION TITLE
Unexpected token 'const' [SOLVED]

### DISCUSSION THREAD

**Discussion by organicnz** [AntlerVC/firetable#986] [Support / Q&A] [answered]
Hi guys :) 

How to fix this error on Rowy `Unexpected token 'const'`? It woked with Food checking before :)

Thank you :)

```
// Checking type of post from different booleans and returning post type string 

const derivative:Derivative = async ({row,ref,db,storage,auth})=>{
    let post_type = String("");
    const post_free_type = row.post_free;
    const post_things_type = row.post_things;
    if (post_free_type == true) {
      post_type = String("Food"); 
    } else if (post_things_type == true) {
      post_type = String("Things"); 
    } else {
      post_type = "n/a";
    }
    return post_type;
}
```

<img width="1440" alt="Screen Shot 2022-11-26 at 21 29 56" src="https://user-images.githubusercontent.com/4765090/204104762-7432a70c-a204-4601-b377-2f78dcbc59cd.png">
<img width="1440" alt="Screen Shot 2022-11-26 at 21 37 54" src="https://user-images.githubusercontent.com/4765090/204104768-0e1fda85-007b-461f-8b5b-4bc967dbb281.png">

**Comment by bofeiw** [comment, marked as answer]
@organicnz  Seems like the comments above the code is causing the error. Can you try removing the comment? I will put up a fix for the comment issue.

**Reply by organicnz** [reply]
@bofeiw ohh, thank you very much for your help, mate :)
